### PR TITLE
Add new company - FRAUDIO

### PR DESCRIPTION
**Pull request by Oduig**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [x] I am an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: company-profiles/fraudio.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Fraudio
+
+## Company blurb
+
+Our startup https://fraudio.com is bringing the latest AI technology to the finance and payments sector and is looking to disrupt it. 
+We use cutting-edge technology and research-driven techniques to create tools that productionize AI for our clients, to combat various illicit activites 
+such as transaction fraud, merchant fraud and money laundering.
```
Comment (dougaitken) [inline company-profiles/fraudio.md:7]: There is an odd line break here.

File: company-profiles/fraudio.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Company size
+
+20-30
+
+## Remote status
+
+We are a fully distributed team and all our daily processes are designed for remote work.
+After COVID, we plan to have regular gettogethers with all of us a few times a year, 
+and likely work together in the office for a few days a month for those living nearby.
```
Comment (dougaitken) [inline company-profiles/fraudio.md:21]: Another odd line break

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/fraudio.md (hunk 1 above)
Comment (Oduig) [reply company-profiles/fraudio.md]: Fair enough! I removed the unnecessary line breaks.

File: company-profiles/fraudio.md (hunk 2 above)
Comment (Oduig) [reply company-profiles/fraudio.md]: Fair enough! I removed the unnecessary line breaks.

**Review by dougaitken — APPROVED** [review]

**Pull request merged**